### PR TITLE
Mturk ratings news

### PR DESCRIPTION
**Pull request by marcelhollerbach**
- Implements sending the messages to the workers
- Refactored the messages out in there own files.

### CODE REVIEW

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/MturkPlatform.java [hunk 1]
```diff
@@ -88,41 +89,32 @@ public Boolean isCalibrationAllowed() {
...
-            }
-        })
-        .filter(s1 -> !s1.isEmpty())
-        .collect(Collectors.toList());
+    private static String loadFiles(String file) {
+        try {
+            return CharStreams
+                    .toString(new InputStreamReader(
+                            Main.class.getResourceAsStream(file)
+                            , Charsets.UTF_8));
```
Comment (LeanderK) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/MturkPlatform.java:101]: normally you would use `Files.readAllLines(Paths.get(path), encoding);`

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/MturkPlatform.java [hunk 2]
```diff
@@ -88,41 +89,32 @@ public Boolean isCalibrationAllowed() {
...
+        params.put("PlatformName", getID());
+        params.put("WorkerServiceUrl", workerServiceUrl);
+        params.put("WorkerUIUrl", workerUIUrl);
+        params.put("ExperimentId", experiment.getId()+"");
+        params.put("JsEmbed", jsContent);
+
+        String content = Template.apply(htmlContent, params);
 
         return new PublishHIT(connection,experiment.getTitle(),experiment.getDescription(),
                 experiment.getPaymentBase().getValue()/100d, //we are getting cents passed and have to pass dallers
```
Comment (LeanderK) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/MturkPlatform.java]: typo....`dallers`

Comment (marcelhollerbach) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/MturkPlatform.java]: Not part of this PR :P

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/MturkPlatform.java [hunk 3]
```diff
@@ -274,12 +273,47 @@ private void verifyConsistence(String id, List<PaymentJob> paymentJobs, Map<Stri
         }
     }
 
+    private CompletableFuture<Boolean> notifyWorker(PaymentJob paymentJob) {
```
Comment (LeanderK) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/MturkPlatform.java]: javadoc would be nice

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/MturkPlatform.java [hunk 4]
```diff
@@ -274,12 +273,47 @@ private void verifyConsistence(String id, List<PaymentJob> paymentJobs, Map<Stri
...
+        int counter = 0;
+        List<CompletableFuture<Boolean>> messages = new ArrayList<>();
+
+        String tooLongMessage = loadFiles("/mturk/TooLongMessage.txt");
+        String subjectLine = loadFiles("/mturk/SubjectLine.txt");
+        //while loop sents messages if needed message would be gibber than MAX_LENGTH
+        while(length - current_location > NotifyWorker.MAX_LENGTH) {
+            int old_current_location = current_location;
+            //the new location is the maximum - the too long message
+            current_location = current_location + (NotifyWorker.MAX_LENGTH - tooLongMessage.length());
```
Comment (LeanderK) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/MturkPlatform.java]: are you checking for message size? what is the location pointing to?

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/MturkPlatform.java [hunk 5]
```diff
@@ -274,12 +273,47 @@ private void verifyConsistence(String id, List<PaymentJob> paymentJobs, Map<Stri
...
+
+            String message = paymentJob.getMessage()
+                    .substring(old_current_location, current_location) + "\n"+tooLongMessage;
```
Comment (LeanderK) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/MturkPlatform.java:303]: why substring?

Comment (marcelhollerbach) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/MturkPlatform.java:303]: Because i need to split the string into parts which are fitting into the message ...

Comment (marcelhollerbach) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/MturkPlatform.java:303]: Or what do you expect /want to see ?

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/MturkPlatform.java (hunk 1 above)
Comment (marcelhollerbach) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/MturkPlatform.java:101]: may I ask why ? in this case it makes the code longer, since a have to join the list of lines to one long line ...

Comment (LeanderK) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/MturkPlatform.java:101]: ok, thats a valid argument ;)

File: src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/MturkPlatform.java (hunk 5 above)
Comment (LeanderK) [inline src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/mturk/MturkPlatform.java:303]: resolved
